PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs

```diff
@@ -29,7 +29,7 @@ public void LoadQuestions_RetrievesQuestionsFromFile()
     [TestMethod]
     [DataRow("1", true)]
     [DataRow("2", false)]
-    public void DisplayResult_ReturnsTrueIfCorrect(string userGuess, bool expectedResult)
+    public void DisplayResult_ReturnsTrue_IfCorrect(string userGuess, bool expectedResult)
```

Comment:
```suggestion
    public void DisplayResult_IfCorrect_ReturnsTrue(string userGuess, bool expectedResult)
```
If adjusting this, I would go one step farther so this actually follows guidelines of SUT_Condition_Result